### PR TITLE
add page for cloud shared responsibility model

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves [#2243](https://github.com/redpanda-data/documentation-private/issues/2243)

Preview: 
https://deploy-preview-316--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/#shared-responsibility-model

### CODE REVIEW

**Comment by Deflaimun** [comment]
Renamed from responsAbility to responsIbility.

**Comment by bpraseed** [comment]
I would compress the firt 2 para to be like:
"The Redpanda Shared responsibility models clarify the expectations around cloud resource ownership between Redpanda and Customer with respect to infrastructure, security and Redpanda service.
"

@toddfarmer - can you review the verbage as well ? TIA

File: modules/reference/pages/cloud-shared-responsibility.adoc
Comment (Deflaimun) [inline modules/reference/pages/cloud-shared-responsibility.adoc]:
```suggestion
Redpanda Cloud shared responsibility model clarify the expectations around cloud resource ownership between Redpanda and Customer with respect to infrastructure, security and Redpanda service.
```

File: modules/reference/pages/cloud-shared-responsibility.adoc
Comment (Feediver1) [inline modules/reference/pages/cloud-shared-responsibility.adoc]:
```suggestion
The Redpanda Cloud shared responsibility model clarifies cloud resource ownership between Redpanda and customers with respect to infrastructure, security, and Redpanda service.
```

File: modules/reference/pages/cloud-shared-responsibility.adoc
Comment (Feediver1) [inline modules/reference/pages/cloud-shared-responsibility.adoc]:
```suggestion
Since BYOC is a fully-managed offering running in customer cloud accounts, a small set of responsibilities are shared. However, Redpanda strives to ensure that the user experience is and remains a smooth SaaS experience.
```

File: modules/reference/pages/cloud-shared-responsibility.adoc
Comment (Feediver1) [inline modules/reference/pages/cloud-shared-responsibility.adoc]:
```suggestion
The following table shows the shared responsibility model across current Redpanda Cloud offerings.
```

**Comment by Feediver1** [comment]
@bpraseed and @c4milo This is a stopgap until @micheleRP returns from PTO next week. If you would like more content around this (from the blog), let's file another issue and have it updated then. For now, does this suffice? thx

**Review by Feediver1 — APPROVED** [review]

**Comment by toddfarmer** [comment]
I understand we're establishing RP ownership of Compute, Local storage and Tiered Storage (FYI, the capitalization differs on "storage") for both BYOC classes. That reflects our role in launching and managing these resources. At the same time, these resources live within the customer's cloud account, so it's hard for me to say we "own" these resources. If the customer choses to terminate our access to the Compute resources, for instance - there's no way we could continue to manage them (AFAIK). They delegate specific management components, but it seems like they "own" those resources.

I'm unclear what "Infrastructure vulnerability remediation" means in this context. Is it something more than "Access controls and audit", referenced separately?

**Comment by Feediver1** [comment]
@bpraseed I thought this was urgent? I was off last week, but just noticed this PR as still open. Doesn't look like we can get it across the finish line until we get some clear guidance on the ownership question posed by @toddfarmer . 
Given Todd's observation about "ownership", would it make more sense to use the term "responsibility" instead--or just not using the term "ownership"? This would seem to resolve the question posed by Todd. Also--is this impacted by Serverless? If so, will there be a separate doc issue to update this content after we release it? And more more thing...(sorry), is there anything from this [blog post](https://redpanda.com/blog/fully-managed-byoc-data-sovereignty) that should be added to this update?

Please advise. Thanks!

**Comment by bpraseed** [comment]
@Feediver1 - added link to updated doc [here](https://github.com/redpanda-data/documentation-private/issues/2243#issuecomment-1967541862).. I dont see that updated yet.

File: modules/reference/pages/cloud-shared-responsibility.adoc
Comment (Feediver1) [inline modules/reference/pages/cloud-shared-responsibility.adoc]:
```suggestion
The Redpanda Cloud shared responsibility model clarifies cloud resource responsibilities between Redpanda and customers with respect to infrastructure, security, and Redpanda service.
```

**Comment by Feediver1** [comment]
In response to @toddfarmer 's feedback, seems like the word "ownership" should be removed from the section and table--use "responsibilities" instead. Awaiting review/approval from @bpraseed .

**Comment by micheleRP** [comment]
@bpraseed @c4milo I moved this into a section in the Cloud Overview page. (Please don't review the rest of the page: it's all getting updated with Serverless next week.) For readability, especially on small devices, I decided to split the table into [separate tabs for Dedicated, BYOC, and BYOC with customer-managed VPC](https://deploy-preview-316--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/#shared-responsibility-model). I hope this is OK, because I expect this will grow again with Serverless. Please let me know if you need any changes to this, or approve if it looks good!

**Comment by micheleRP** [comment]
@bpraseed, @c4milo please confirm if [this ](https://deploy-preview-316--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/?tab=tabs-1-dedicated) shows the correct info for doc. The gsheet shows a separate version for marketing.

**Comment by bpraseed** [comment]
@micheleRP - @toddfarmer suggested "Responsibility Ownership" in the original sheet link. Can we incorporate that ? Also add @toddfarmer and @tmgstevens as approvals for merging this and remove Camilo. TIA for looking into this.

**Review by bpraseed — APPROVED** [review]

**Pull request merged**
